Formula/wp-cli.rb

```diff
@@ -0,0 +1,16 @@
+class WpCli < Formula
+  desc "Command-line interface for WordPress"
+  homepage "https://wp-cli.org"
+  url "https://github.com/wp-cli/wp-cli/releases/download/v1.5.0/wp-cli-1.5.0.phar"
+  sha256 "f615d57957e66a09f57acc844a1fc5402e9fa581dcb387bbe1affc4d155baf9d"
+
+  bottle :unneeded
+
+  def install
+    bin.install "wp-cli-#{version}.phar" => "wp"
+  end
+
+  test do
+    system "#{bin}/wp", "--info"
```

Comment:
Can you include a test that exercises some of the software's functionality? Like get the status of a site, or try an operation without proper credentials and check for failure?

Reply:
Yes I'll do that.